test/logisticalTests.js

```diff
@@ -102,11 +171,14 @@ describe('Logistical', function(){
         assert.equal(result, subject.classify(test));
       }
     });
-    it('correctly classifies large test data', function(){
+    */
+
+    it('correctly classifies large test data');
+    /*, function(){
```

Comment:
Instead of commenting out tests, I would rather use it.skip() which will have it skip the test. Commented out code always concerns me since it's not clear if it is a placeholder, if it doesn't work yet or if it works but we don't want to run it for some reason. it.skip means the test can stay in place but mocha won't run it.